src/main/scala/org/apache/spark/sql/execution/datasources/greenplum/GreenplumUtils.scala

```diff
@@ -106,8 +106,13 @@ object GreenplumUtils extends Logging {
       schema: StructType,
       options: GreenplumOptions): Unit = {
     val randomString = UUID.randomUUID().toString.filterNot(_ == '-')
+    val canonicalTblName = TableNameExtractor.extract(options.table)
+    val schemaPrefix = canonicalTblName.schema.map(_ + ".").getOrElse("")
```

Comment:
why the schema name is in the `options.table`, why  do we left `options.dbschema` behind.

Reply:
user may set options.table, which is extended from JDBCOptions and  can not be modify,  with schema directly.